tests/reliability/test_cluster/test_cluster_logs/test_cluster_worker_logs_order/data/Agent-groups_recv.yaml

```diff
@@ -0,0 +1,71 @@
+# Root
+- log_id: root
+  parent: null
+  tag: Agent-groups recv root
+
+# Common logs
+- log_id: log0
+  parent: root
+  tag: Starting.*
+- log_id: log1
+  parent: log0
+  tag: .*chunks updated in wazuh-db in.*
+- log_id: log2
+  parent: log1
+  tag: Obtained.*chunks of data in.*
+
+# No need to synchronize after failed synchronization tries.
+- log_id: log3
+  parent: log2
+  tag: The checksum of both databases match.*Reset the attempt counter.
+- log_id: log4
+  parent: log3
+  tag: Finished in.*Updated.*chunks.*
+
+# No need to synchronize.
+- log_id: log15
+  parent: log2
+  tag: The checksum of both databases match.*
+- log_id: log16
+  parent: log15
+  tag: Finished in.*Updated.*chunks.*
+
+# Forced synchronization is required.
+- log_id: log5
+  parent: log2
+  tag: Checksum comparison failed. Attempt 10/10.*
+- log_id: log7
+  parent: log5
+  tag: Sent request to obtain all agent-groups information from the master node.*
+- log_id: log8
+  parent: log7
+  tag: Finished in.*Updated.*chunks.*
+
+# Checksum comparison fails.
+- log_id: log12
+  parent: log2
+  tag: Checksum comparison failed. Attempt .*
+- log_id: log13
+  parent: log12
+  tag: Finished in.*Updated.*chunks.*
+
+# The checksum differs
+- log_id: log9
+  parent: log2
+  tag: The checksum of master (.*) and worker (.*) are different.
+
+# The checksums differ and the agent_groups_mismatch_limit was not exceeded
+- log_id: log17
+  parent: log9
+  tag: Checksum comparison failed.*
+- log_id: log18
+  parent: log17
+  tag: Finished in.*Updated.*chunks.*
+
+# The checksums differ and the agent_groups_mismatch_limit was exceeded
+- log_id: log10
+  parent: log9
+  tag: Sent request to obtain all agent-groups information from the master node.*
+- log_id: log11
+  parent: log10
+  tag: Finished in.*Updated.*chunks.*
```

Comment:
This is the same as log7 and log8

Reply:
## Update

Updated log messages:

https://github.com/wazuh/wazuh-qa/blob/fad11f30da4a8b68ff7a1e8e9bbafc630318aff4/tests/reliability/test_cluster/test_cluster_logs/test_cluster_worker_logs_order/data/Agent-groups_recv.yaml#L1-L55